src/components/customer-resource-show.js

```diff
@@ -141,7 +138,13 @@ export default class CustomerResourceShow extends Component {
           model={model}
           paneTitle={model.name}
           paneSub={model.packageName}
-          actionMenuItems={actionMenuItems}
+          lastMenu={(
+            <PaneMenu>
+              <Link to={`/eholdings/customer-resources/${model.id}/edit`}>
+                <IconButton icon="edit" />
```

Comment:
I think this renders an actual `<button>` within the `<a>` (and I think we might be doing this elsewhere?). I just checked and `IconButton` accepts an `href` prop and it will use `Link` behind the scenes. This way we can avoid the accessibility and browser issues with nested interaction elements.